src/features/components/templates/navigation/main-nav/content/resources/data.ts

```diff
@@ -0,0 +1,64 @@
+import { TSmartNavContent } from '../../../types'
+import Blog from 'images/svg/custom/blog-nav.svg'
+import Community from 'images/svg/menu/community.svg'
+import Help from 'images/svg/menu/help-center.svg'
+import Payment from 'images/svg/menu/payment-methods.svg'
+import Signals from 'images/svg/menu/signals.svg'
+
+export const resourcesNavSectionContent: TSmartNavContent[] = [
+    {
+        id: 0,
+        data: {
+            icon: {
+                src: Help,
```

Comment:
Any chance to use StaticImage in this kind of files? Should work
I already used this approach here: https://github.com/binary-com/sinbad/blob/master/src/pages/home/welcome.tsx
Please check and let us know what are you thinking about it 🙏